verl/trainer/ppo/ray_trainer.py

```diff
@@ -911,13 +913,21 @@ def _save_checkpoint(self):
         )
 
         print(f"local_global_step_folder: {local_global_step_folder}")
-        actor_local_path = os.path.join(local_global_step_folder, "actor")
-
-        actor_remote_path = (
+        s3_global_step_folder = (
```

Comment:
let's set self.default_remote_dir in trainer.__init__ so that this part of the code is simpler